src/app/workflow/dag/dag.component.html

```diff
@@ -41,15 +49,18 @@
               <mat-icon id="resize-full-button" *ngIf="!expanded">fullscreen</mat-icon>
               <mat-icon id="resize-small-button" *ngIf="expanded">fullscreen_exit</mat-icon>
             </button>
-            <button id="dag_reload" (click)="reset()" class="dag-button btn btn-default" matTooltip="Reset">
+            <button *ngIf="dagType !== 'wdlviewer'" id="dag_reload" (click)="reset()" class="dag-button btn btn-default" matTooltip="Reset">
+              <mat-icon>control_camera</mat-icon>
+            </button>
+            <button *ngIf="dagType === 'wdlviewer' && enableWdlViewer" (click)="resetPipelineBuilder()" class="dag-button btn btn-default" matTooltip="Reset">
```

Comment:
Last comment on `enableWdlViewer`. :) If there are other usages, your removing it from dockstore.model.ts should cause compile errors.